LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020, NumPy Community.
+Copyright (c) 2024, NumPy Community.
```

Comment:
Great catch! Do we want to give the range here like the [numpy documentation](https://numpy.org/devdocs/) (i.e. something like `2020 - 2024` - see the footer of the numpy docs for the example)!

I have no strong opinions so I leave it up to you!

Reply:
@rossbar Sounds good!